packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml

```
@@ -1146,6 +1158,73 @@ spec:
             content: {{or $proxy $image}}
             encoding: base64
             permissions: "0444"
+  - name: apiserverEncryptionConf
+    definitions:
+      - selector:
+          apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+          kind: KubeadmControlPlaneTemplate
+          matchResources:
+            controlPlane: true
+          jsonPatches:
+            - op: add
+              path: /spec/template/spec/kubeadmConfigSpec/files
+              valueFrom:
+                template: |
+                  - path: /etc/kubernetes/encryption-config.yaml
```

```suggestion
              path: /spec/template/spec/kubeadmConfigSpec/files/-
              valueFrom:
                template: |
                    path: /etc/kubernetes/encryption-config.yaml
```

```suggestion
              path: /spec/template/spec/kubeadmConfigSpec/files/-
```

This needs to append to files, to not replace already existing files.